### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by nneveu**
# Summary of changes

Renamed `data_analysis` folder to `data`, and updated tests to use new name. Also removed old fitting file that was replaced by recent PR.

# To Test
The standard workflow checks should suffice.

### CODE REVIEW

**Comment by nneveu** [comment]
I'd like to add a more general YAML file reference so I'm putting this in draft.

**Review by lisazacarias — APPROVED** [review]

File: lcls_tools/common/devices/reader.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 from lcls_tools.common.devices.area import Area
 from lcls_tools.common.devices.beampath import Beampath
 
-
-DEFAULT_YAML_LOCATION = "./lcls_tools/common/devices/yaml/"
+DEFAULT_YAML_LOCATION = os.path.join(os.path.dirname(__file__), "yaml/")
```
Comment (MattKing06) [inline lcls_tools/common/devices/reader.py:11]: This is a good change, allows people to set `lcls_tools.common.devices.reader.DEFAULT_YAML_LOCATION` to a different location in their scripts if that's needed too!

**Comment by nneveu** [comment]
@lisazacarias or @MattKing06 could I get another quick review?

**Comment by MattKing06** [comment]
All looks good to me, didn't see the need to run anything since it was just module renaming and formatting!

**Review by MattKing06 — APPROVED** [review]

**Pull request merged**